install.md

```diff
@@ -49,6 +49,10 @@ follow the OS-specific instructions to install pip if you don't know how to do i
 
       sudo apt-get install python-pip
 
+- **Arch / Manjaro (Python3 distributions)**:
```

Comment:
Or just changing the title, instead of "Arch / Manjaro (Python3 distributions)", something like "Python 3 linux distributions (like archlinux)".